ppb/systems.py

```diff
@@ -26,9 +33,19 @@ class PygameEventPoller(System):
     An event poller that converts Pygame events into PPB events.
     """
 
-    event_map = {
-        pygame.QUIT: events.Quit,
-    }
+    event_map = None
+
+    def __new__(cls, *args, **kwargs):
+        if cls.event_map is None:
+            cls.event_map = {
+                pygame.QUIT: "quit",
+                pygame.MOUSEMOTION: "mouse_motion"
```

Comment:
nitpick! ending every collection entry with a comma, including the last one, makes for cleaner diffs later (so you don't need to edit a line just to add a comma)